src/js/paackSvgIconSprite.js

```diff
@@ -11,5 +11,7 @@ class PaackSvgIconSprite extends HTMLElement {
     this.innerHTML = svg;
   }
 }
+if(typeof window === 'undefined'){
+  window.customElements.define('paack-svg-icon-sprite', PaackSvgIconSprite);
+}
```

Comment:
What do you think to export function to define this Web Component like this; 
```js
const definePaackSvgIconSprit = () => window.customElements.define('paack-svg-icon-sprite', PaackSvgIconSprite);
```
I think it gives us more flexibility to handle/support SSR, ISR, SSG ... outside of this module [ELM/React].

Also, we should update the condition of the current if statement;

```suggestion
if(typeof window !== 'undefined') window.customElements.define('paack-svg-icon-sprite', PaackSvgIconSprite);
```

Reply:
for me, that's fine.

Reply:
Sounds good for me...

Reply:
@3imed-jaberi I have updated missing not(!) 😓